src/controllers/user.controller.ts

```diff
@@ -29,15 +29,16 @@ export const getAllUsersHandler = async (
 	res: Response,
 ) => {
 	try {
-		let { page, size } = req.query;
+		let { page, size, receiveMarketingEmails } = req.query;
 
 		if (!page) page = "1";
 		if (!size) size = "10";
 
 		const limit = +size;
 		const skip = (+page - 1) * limit;
+		const query = receiveMarketingEmails ? { receiveMarketingEmails } : {};
```

Comment:
The case of this field in the database is changed, please look at that

Reply:
Thanks for pointing it , I just overlooked it , sorry. I will make the changes.